player/command.c

```diff
@@ -3760,6 +3927,7 @@ static const struct m_property mp_properties_base[] = {
     {"input-bindings", mp_property_bindings},
 
     {"shared-script-properties", mp_property_script_props},
+    {"udata", mp_property_udata},
```

Comment:
my bikeshed contribution: either `userdata` or `user` but abbrevations are meh

Reply:
> my bikeshed contribution: either `userdata` or `user` but abbrevations are meh

`udata` is just plain bad, not a bikeshed at all.

Reply:
Just recording my bikeshed here, but I prefer `userdata`. It's very clear to me what that does as opposed to `udata` which makes me think of μ like utorrent or something.